ethereum-testnets.asciidoc

```diff
@@ -2,12 +2,31 @@
 
 === What is testnet?
 
+A test network is used to simulate behavior of ethereum network. There are some publicly available test networks.
```

Comment:
I would add articles. Maybe: simulate THE behavior of THE main ethereum network and some variations.